android/build_boinc_arm.sh

```diff
@@ -23,14 +23,26 @@ export TOOLCHAINROOT="$NDK_ROOT/toolchains/llvm/prebuilt/linux-x86_64/"
 export TCBINARIES="$TOOLCHAINROOT/bin"
 export TCINCLUDES="$ANDROIDTC/arm-linux-androideabi"
 export TCSYSROOT="$TOOLCHAINROOT/sysroot"
+export VCPKG_DIR=$VCPKG_ROOT/installed/arm-android
+
+CONFIG_FLAGS=""
+VCPKG_LDFLAG=""
+
+if [ $BUILD_WITH_VCPKG = "yes" ]; then
+    VCPKG_LDFLAG="-L$VCPKG_DIR/lib"
+    CONFIG_FLAGS="--with-libcurl=$VCPKG_DIR --with-ssl=$VCPKG_DIR --enable-vcpkg"
+else
+    CONFIG_FLAGS="--with-ssl=$TCINCLUDES"
+    VCPKG_LDFLAG="-L$TCSYSROOT/usr/lib -L$TCINCLUDES/lib"
```

Comment:
The same applicable to this variable: `VCPKG_LDFLAG`
Maybe it's better to rename to `CONFIG_LDFLAGS`?

Reply:
fix